src/lib/eZ/RichText/ValidatorInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformRichText\eZ\RichText;
+
+use DOMDocument;
+
+interface ValidatorInterface
```

Comment:
Could we use this opportunity to move all Validators, **except this interface**. to a separate namespace `EzSystems\EzPlatformRichText\eZ\RichText\Validator`?